syscalls_windows.go

```diff
@@ -170,12 +170,33 @@ func getdeviceid(componentID string) (deviceid string, err error) {
 				key.Close()
 				continue
 			}
+			if len(interfaceName) > 0 {
+				key2 := `SYSTEM\CurrentControlSet\Control\Network\{4D36E972-E325-11CE-BFC1-08002BE10318}\`+val+`\Connection`
```

Comment:
The `SYSTEM...318}` part is also used above. Could probably make it a const and reuse?